File: xFit/xFit.Model/Requests/KorisnikInsertRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace xFit.Model.Requests
{
	public class KorisnikInsertRequest
	{
		
		public string? Ime { get; set; }

		public string? Prezime { get; set; }

		public DateTime? DatumRodjenja { get; set; }

		public string? KorisnickoIme { get; set; }

		[Compare("PasswordPotvrda",ErrorMessage ="Passwords do not match. ")]
		public string? Password { get; set; }

		[Compare("Password", ErrorMessage = "Passwords do not match. ")]
		public string? PasswordPotvrda { get; set; }
		public int? GradId { get; set; }
		public int? SpolId { get; set; }
	}
}

File: xFit/UI/xfit_mobilna/lib/models/korisnikInsertRequest.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnikInsertRequest.g.dart';

@JsonSerializable()
class KorisnikInsertRequest {
  String? ime;
  String? prezime;
  DateTime? datumRodjenja;
  String? korisnickoIme;
  String? password;
  String? passwordPotvrda;
  int? gradId;
  int? spolId;

  KorisnikInsertRequest(this.ime, this.prezime, this.datumRodjenja, this.korisnickoIme,this.password, this.passwordPotvrda, this.gradId,this.spolId);

  factory KorisnikInsertRequest.fromJson(Map<String, dynamic> json) => _$KorisnikInsertRequestFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikInsertRequestToJson(this);
}
